src/commands/dev/lib/dev-server.ts

```diff
@@ -392,36 +398,34 @@ export default class DevServer {
     ]);
     this.env = env;
     this.buildEnv = buildEnv;
-    const nowJson = await this.getNowJson();
 
-    if (nowJson) {
-      const builders = (nowJson.builds || []).map((b: BuildConfig) => b.use);
-      const shouldUpdate = true;
-      await installBuilders(builders, shouldUpdate);
-      await this.updateBuildMatches(nowJson);
+    const nowJson = await this.getNowJson();
```

Comment:
Can this line be moved into the Promise.all above so that it gets the 3 files concurrently?

Reply:
Or is that going to hide errors if it throws?